ajax/libs/modern-normalize/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "modern-normalize",
+  "description": "Normalize browsers' default style",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/sindresorhus/modern-normalize.git"
+  },
+  "author": {
```

Comment:
The author field should be `name <email> (url)`

Reply:
@Billy4195 Done :)